pennylane/ops/qubit/state_preparation.py

```diff
@@ -127,9 +129,15 @@ def state_vector(self, wire_order: Optional[WiresLike] = None) -> TensorLike:
 
 
 class StatePrep(StatePrepBase):
-    r"""StatePrep(state, wires)
+    r"""StatePrep(state, wires, pad_with, nomralize)
```

Comment:
```suggestion
    r"""StatePrep(state, wires, pad_with, normalize)
```

This also misses out `default` values in the documentation.